dbms/src/Storages/KVStore/tests/gtest_region_persister.cpp

```diff
@@ -199,7 +203,7 @@ TEST_F(RegionSeriTest, RegionOldFormatVersion)
 try
 {
     TableID table_id = 100;
-    auto region = std::make_shared<Region>(createRegionMeta(1001, table_id));
+    auto region = makeTmpRegion();
```

Comment:
The `makeTmpRegion()` create a region with `table_id == 1` but not the `table_id` here. It may cause test failure?

Reply:
No, in this test suite, we only focus on KVStore, so we have no schema.

Reply:
OK